python/test/test_model.py

```diff
@@ -0,0 +1,55 @@
+import starwrap as sw
+import numpy as np
+import os
+import pytest
+
+
+@pytest.fixture
+def sp():
+    arg = sw.args()
+    arg.trainFile = "./input.txt"
+    arg.testFile = "./input.txt"
+    arg.trainMode = 5
+
+    sp = sw.starSpace(arg)
+    sp.init()
+    sp.train()
+    # sp.evaluate()
+
+    sp.nearestNeighbor("some text", 10)
+
+    yield sp
+
+
+def test_train_model(sp):
+
+    result1 = np.array(sp.getDocVector("this\tis\ttest", "\t"))
+    result2 = np.array(sp.getDocVector("this is test", " "))
+
+    print(f"result1: ${result1}")
+    print(f"result2: ${result2}")
+
+    assert (result1 == result2).all()
```

Comment:
is this tested? `( a == b )` produces a bool, which has no attribute all()

Reply:
Yes, it was. Seems like Numpy related stuff. https://www.geeksforgeeks.org/how-to-compare-two-numpy-arrays/

Reply:
ok this is a bit confusing.